templates/miq-template-ext-db.yaml

```diff
@@ -20,6 +20,7 @@ objects:
   stringData:
     pg-password: "${DATABASE_PASSWORD}"
     database_url: "postgresql://${DATABASE_USER}:${DATABASE_PASSWORD}@${DATABASE_SERVICE_NAME}:${DATABASE_PORT}/${DATABASE_NAME}?encoding=utf8&pool=5&wait_timeout=5"
+    v2-key: "${V2_KEY}"
```

Comment:
Given the issues we had with this, I would be more likely to move `database_url` to `database-url`.

Sound good?